react-app/ccf/src/components/applications/AdminCoverPageModal.tsx

```
@@ -0,0 +1,72 @@
+    const researchCoverPage = (
+      <div className="cover-page-modal-child">
+        <div className="funding-info">{application.decision}</div>
+            <div className="application-pdf-link">{pdfLink ? <a target="_blank" href={pdfLink.url}>Application PDF</a>: ""}</div>
+            <div className="post-grant-report-pdf-link">{reportLink ? <a target="_blank" href={reportLink.url}>Post Grant Report</a>: reportMsg}</div>
+        <Review type={application.grantType} formData={application} hideFile={true}></Review>
+      </div>
+    )
+
+    
+    return (
+      <Modal isOpen={isOpen} onClose={onClose} children={researchCoverPage} fullscreen={true}></Modal>
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)